tests/phpunit/elementor/modules/atomic-widgets/dynamic-tags/test-dynamic-prop-type.php

```diff
@@ -37,54 +37,76 @@ public function tearDown(): void {
 		Plugin::$instance->dynamic_tags = $this->original_dynamic_tags;
 	}
 
```

Comment:
Don't forget to add tests as we discussed

Reply:
Done